src/translations/indexMapper.js

```diff
@@ -9,22 +9,45 @@ import { mixin } from '../helpers/object';
 import { isDefined } from '../helpers/mixed';
 
 /**
- * Index mapper manages the mappings provided by "smaller" maps called index map(s). Those maps provide links from
- * indexes (physical¹ or visual² depending on requirements) to another value. For example, we may link physical column
- * indexes with widths of columns. On every performed CRUD action such as insert column, move column and remove column
- * the value (column width) will stick to the proper index. The index mapper is used as the centralised source of truth
- * regarding row and column indexes (their sequence, information if they are skipped in the process of rendering,
- * values linked to them). It handles CRUD operations on indexes and translate the visual indexes to the physical
- * indexes and the other way round³. It has built in cache. Thus, this way, read operations are as fast as possible.
- * Cache updates are triggered only when the data or structure changes.
+ * Index mapper stores, registers and manages the indexes on the basis of calculations collected from this class objects.
+ * It should be seen as a single source of truth for any operation that considers CRUD actions such as **insertion**, **movement**, **removal** etc, and is used to properly calculate physical and visual indexes translations in both ways.
  *
- * ¹ Physical index is particular index from the sequence of indexes assigned to the data source rows / columns
- * (from 0 to n, where n is number of the cells on the axis).
- * ² Visual index is particular index from the sequence of indexes assigned to visible rows / columns
- * (from 0 to n, where n is number of the cells on the axis).
- * ³ It maps from visible row / column to its representation in the data source and the other way round.
- * For example, when we sorted data, our 1st visible row can represent 4th row from the original source data,
- * 2nd can be mapped to 3rd, 3rd to 2nd, etc. (keep in mind that indexes are represent from the zero).
+ * To ensure the calculation is done correctly **index mapper** also registers the skipping process
+ * and embeds **cache** which is triggered only when the data or structure changes.
+ *
+ *
+ * Index mapper can be called upon specific properties i.e. **rowIndexMapper**, **columnIndexMapper** which, respectively, manage **rows** and **columns**.
+ *
+ * **There are 3  main objects on class IndexMapper for storing different types of index information:**
+ *
+ * **-** Map for the **sequence** of indexes. It is registered by default and may be used from API methods like
+ * `getIndexesSequence`, `setIndexesSequence`, `getNumberOfIndexes` etc.
+ * ```js
+ * // type {VisualIndexToPhysicalIndexMap}
+ * this.indexesSequence = new IndexToIndexMap();
+ *
+ * ```
+ * **-** Collection for different **skip maps**. These indexes are bound to be skipped and will not be rendered.
```

Comment:
True. But only half of it, skip maps are not about rendering only. Skipped indexes will not be renderer AND will not be a part of `getData` calls. They will not be exported to file, or copied to clipboard.

As mentioned above, there will be added `hiddenMapsCollection` which will be for rendering only. Indexes in the hidden map will not be rendered but will be included in `getData` calls. They will be exported to file and copied to clipboard.

Reply:
Thank you, I will include the missing part that then :)

"`hiddenMapsCollection`  will be added" ? When and where? I looked here https://github.com/handsontable/handsontable/issues/5751 but could not find it.

Reply:
Where: #6178 
When: 🙄

Reply:
thanks, no pressure intended 🙈!